src/vunnel/providers/sles/parser.py

```diff
@@ -142,6 +142,10 @@ def _get_name_and_version_from_test(
 
         name = name_obj.name
         version = version_obj.value
+        if version_obj.operation == "greater than" and version_obj.value == "0:0-0":
```

Comment:
The assumption here is this: That packages that are vulnerable in a given version of SLES, if they are affected and no patch is available, are considered vulnerable if present.

In other words, saying "PackageA is vulnerable" is always either: "All versions of PackagA are vulnerable" or saying, "PackageA is vulnerable until version 0:1.23", and we never say something like, "PackageA is vulnerable for versions greater than 0:1.23 and less than 0:2.0" or something.

Is this assumption valid @BenoitGui or @msmeissn ?

Reply:
It seems valid for me, but @msmeissn should confirm this assumption since he is the author of the code 👍